src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.h

```diff
@@ -7,9 +7,25 @@ struct EnElfgrp;
 
 typedef void (*EnElfgrpActionFunc)(struct EnElfgrp*, GlobalContext*);
 
+#define ENELFGRP_GET(thisx) ((thisx)->params & 0xF)
+#define ENELFGRP_GET_FE00(thisx) (((thisx)->params & 0xFE00) >> 9)
+
+enum {
+    /* 0 */ ENELFGRP_0,
+    /* 1 */ ENELFGRP_1,
+    /* 2 */ ENELFGRP_2,
+    /* 3 */ ENELFGRP_3,
+    /* 4 */ ENELFGRP_4,
+    /* 8 */ ENELFGRP_8 = 8,
```

Comment:
Now you shouldn't need this 8, since it was only used to represent `STRAY_FAIRY_TYPE_TURN_IN_TO_FAIRY_FOUNTAIN`.